tests/unit/xoopsLib/Xoops/Core/Cache/AccessTest.php

```diff
@@ -1,8 +1,8 @@
 <?php
-namespace Xoops\Core\Cache;
```

Comment:
Le 24/09/2015 19:13, Richard Griffith a écrit :
> In tests/unit/xoopsLib/Xoops/Core/Cache/AccessTest.php
> <https://github.com/XOOPS/XoopsCore/pull/404#discussion_r40345105>:
>
>> @@ -1,8 +1,8 @@
>>  <?php
>> -namespace Xoops\Core\Cache;
>
> Why are we removing these from namespaces? Most projects seem to use
> namespaced tests. We have already had conflicts with the same class name
> used in several test classes. Using the natural namespace seems like a
> good idea, and standard practice.
>
> Are you having issues when using them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/XOOPS/XoopsCore/pull/404/files#r40345105>.
>


Richard

I'm inclined to think that namespace are reserved for the core core.

a test file is like an ordinary script (or any other script like modules 
and so on)

For me, unit test have to be as neutral as possible with Core namespaces.
But it is only my own view.